Automattic/wp-calypso
# Site Settings: Add podcast settings page

This PR adds a new route for managing podcast settings. This PR is intentionally incomplete. It's part of a larger epic and behind the feature flag `manage/site-settings/podcasting`. See p3Ex-32D-p2.

![image](https://user-images.githubusercontent.com/363749/37740655-828e8d00-2d2c-11e8-8ab4-e5b3459bdada.png)

:warning: **Note: This PR only includes the basic setting fields. Additional fields such as feed category and podcast image will be added in a future PR. Your podcast may not be "enabled" if you do not have a feed category selected. You can do that from wp-admin in Settings > Media**

To test:
* Sandbox the API and apply D10933-code.
* Directly visit `http://calypso.localhost:3000/settings/podcasting/:site_id`, replacing `:site_id` with your domain slug.
* Verify that settings are loaded and appear as they do in Media settings in wp-admin.
* Verify that changing the settings and then attempting to navigate away from the page prompts you about unsaved changes.
* Verify that any changes are persisted if saved.
